site/en/docs/extensions/mv3/getstarted/extensions-101/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Extensions 101'
+subhead: 'Explore basic concepts of Chrome extension development.'
+description: 'Explore basic concepts of Chrome extension development.'
+date: 2022-08-10
+# updated: 2022-06-13
+---
+
+## Overview {: #overview }
+
+Chrome extensions enhance the browsing experience by adding features and functionality to the Chrome
+browser. This page introduces Chrome extension development and provides links to step-by-step
+[beginner tutorials][section-tutorials]. 
+
+## Web technologies {: #web-tech }
+
+Extensions are written with the same web technologies used to create web applications:
+
+- [HTML][mdn-html] is used as a content markup language.
+- [CSS][web-dev-css] is used for styling.
+- [JavaScript][mdn-js] is used for scripting and logic.
+- Extensions can use all the [JavaScript APIs](https://developer.mozilla.org/docs/Web/API) that the
+  browser provides.
```

Comment:
This doesn't seem like it fits. Rather than describing a distinct core web technology like the first three bullets, it provides additional context on another bullet. It might be better to move this into the paragraph that follows.

Reply:
I'll move it below and see how it flows...✍️